tpm/aziot-tpm-sys/azure-iot-hsm-c/src/hsm_log.c

```diff
@@ -0,0 +1,96 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <time.h>
+
+#include "hsm_log.h"
+
+#if defined __WINDOWS__ || defined _WIN32 || defined _WIN64 || defined _Windows
```

Comment:
There's no Windows support in this repo.

Reply:
Good catch. I thought I got everything windows related, but looks like I missed a spot. Will fix.